ci/testdata/Dockerfile

```diff
@@ -3,6 +3,7 @@ FROM golang:1.19-alpine
 # Install bash and git (and build-base to get gcc)
```

Comment:
While we are updating the `Dockerfile`s, let's update also the alpine version on Line 1 above. Not sure if DeepSource will complain if we just say `golang:alpine` ... if so, we can do `golang:1.24-alpine` I think.